pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/buffer/TransactionBufferClientTest.java

```diff
@@ -161,8 +175,40 @@ public void testAbortOnSubscription() throws ExecutionException, InterruptedExce
             futures.add(tbClient.abortTxnOnSubscription(topic, "test", 1L, i, -1L));
         }
         for (int i = 0; i < futures.size(); i++) {
-            Assert.assertEquals(futures.get(i).get().getMostSigBits(), 1L);
-            Assert.assertEquals(futures.get(i).get().getLeastSigBits(), i);
+            assertEquals(futures.get(i).get().getMostSigBits(), 1L);
+            assertEquals(futures.get(i).get().getLeastSigBits(), i);
+        }
+    }
+
+    @Test
+    public void testTransactionBufferClientTimeout() throws Exception {
+        ConnectionPool connectionPool = mock(ConnectionPool.class);
+        NamespaceService namespaceService = mock(NamespaceService.class);
+        HashedWheelTimer hashedWheelTimer = new HashedWheelTimer();
```

Comment:
Use Lombok's `@Cleanup("stop")` to make sure that the timer is closed when the test completes. This test will "leak" threads if the timer isn't stopped.

Reply:
thanks for your review, i will fix it.